client/my-sites/customer-home/cards/features/webinar-promotion/index.jsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import ExternalLink from 'components/external-link';
+import temporaryIllustration from 'assets/images/customer-home/illustration--task-connect-social-accounts.svg';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const WebinarPromotion = () => {
```

Comment:
I think we can go with just `Webinars`, and `/cards/education` would likely be a better directory fit.

Reply:
Hm, or maybe in `/cards/tasks`. I don't really like how we're convoluting "category of card" with "location of card", but we'll get that sorted.

Reply:
I think it makes sense to move it to `/cards/education` since this is informing a user about something - switched it there! :)